packages/SwingSet/src/message.js

```diff
@@ -151,6 +151,15 @@ export function insistVatSyscallObject(vso) {
       assert.typeof(data, 'string');
       break;
     }
+    case 'vatstoreGetAfter': {
```

Comment:
please also update the typescript definition in src/types.js around line 112, it should get a new `VatSyscallGetAfter` definition, and the `VatSyscallObject` union should get a new member. This will help nail down the fact that the thing argument is `string | undefined`, which will probably reveal some undefined-vs-null-vs-JSON issues.

Reply:
Interesting that TypeScript didn't complain about the absence of this...

Reply:
Also interesting: after I put in the additional type defs TypeScript is still happy.  This may be the first time I've experienced having TypeScript be happy on the first try.  I'm sure I must have done it wrong.

Reply:
It's entirely possible we don't have TS enabled in enough places yet.. `message.js` was mostly written to check the kernel-worker pathways like `manager-helper.js` and `supervisor-subprocess-xsnap.js`. But obviously we should enable it in more and more files over time.